src/main/resources/site/parts/contact/contact.es6

```diff
@@ -1,12 +1,16 @@
 const {
-  getComponent
+  getContent, getComponent
 } = __non_webpack_require__( '/lib/xp/portal')
 const {
   render
 } = __non_webpack_require__( '/lib/thymeleaf')
 const {
   renderError
 } = __non_webpack_require__( '/lib/error/error')
+const { getContactsFromRepo } = __non_webpack_require__('/lib/repo/statreg/contacts')
+const { ensureArray } = __non_webpack_require__('/lib/polyfills/xp-util')
+const { chunkArray } = __non_webpack_require__('/lib/arrayUtils')
```

Comment:
Should look into streamlining utils? We already have a /ssb/utils, and now we get two more. Is that really necessary?

Reply:
I am for **streamlining** utils, but not putting them all in one `utils.es6` -- file just to keep them modular and only-related-utils-in-one-place. Agree that ensureArray and chunkArray might be good to be in one place - will club them together and move them to `/lib/ssb`